code/mob/living/critter/small_animal.dm

```diff
@@ -1453,11 +1455,12 @@ var/list/mob_bird_species = list("smallowl" = /mob/living/critter/small_animal/b
 		src.UpdateOverlays(overlay_eyes, "eyes")
 
 	death()
-		src.ClearAllOverlays()
-		//appears to be missing a dead colorkey
-		//var/image/overlay = image('icons/misc/critter.dmi', "roach_colorkey-dead")
-		//overlay.color = fur_color
-		//src.UpdateOverlays(overlay, "hair")
+		can_lie = 0
+		if (fur_color != 0)
+			src.ClearAllOverlays()
```

Comment:
I know this was here before, but what overlays is this clearing? Seems like it could just randomly remove other overlays which happen to be applied to the roach, like stickers or whatever.

Reply:
Doubt it removes much, its gone now.